app/services/auth/Auth.js

```diff
@@ -0,0 +1,70 @@
+import Auth0Lock from 'auth0-lock';
+import { AUTH_CONFIG } from './auth0-config';
+import history from './history';
+
+export default class Auth {
+  constructor() {
+    this.login = this.login.bind(this);
```

Comment:
If you define your class methods like `login: () => {}` instead of `login() {}` you won't have to bind `this` to these functions. ES6 classes kinda suck